### ISSUE TITLE
Invalid Declaration

### ISSUE THREAD

**Issue by wassimseif** [RxSwiftCommunity/RxAlamofire#55]
Guys i'm still getting the invalid declaration error 
![screen shot 2016-10-12 at 11 39 49 pm](https://cloud.githubusercontent.com/assets/7282773/19326931/369222b2-90d5-11e6-9009-2332a363431d.png)
My podfile 
```
    # Uncomment this line to define a global platform for your project 
 platform :ios, '10.0'

target 'TechStack' do
  # Comment this line if you're not using Swift and don't want to use dynamic frameworks
  use_frameworks!

  # Pods for TechStack
    pod 'ObjectMapper', '~> 2.1'
    pod 'Moya', '8.0.0-beta.2'
    pod 'RxAlamofire', :git => 'https://github.com/RxSwiftCommunity/RxAlamofire.git', :branch => 'develop'

  target 'TechStackTests' do
    inherit! :search_paths
    # Pods for testing
  end

  target 'TechStackUITests' do
    inherit! :search_paths
    # Pods for testing
  end
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['SWIFT_VERSION'] = '3.0'
    end
  end
end

end
```

**Comment by wassimseif** [comment]
it builds when i comment out these to lines and `import RxCocoa`

**Comment by pabsanmez** [comment]
Probably Moya', '8.0.0-beta.2' is downloading RxSwift and RxCocoa 3.0.0-beta.2 and RxAlamofire is not working with beta.2, they have to update it. You can use a previous version of Moya until they upgrade RxAlamofire to beta.2

**Comment by wassimseif** [comment]
This was fixed in #56. Closing now
